src/coreclr/jit/CMakeLists.txt

```diff
@@ -91,7 +91,9 @@ function(create_standalone_jit)
     # We'll be linking against LLVM built without /guard:ehcont, so disable it.
     set_target_properties(${TARGETDETAILS_TARGET} PROPERTIES CLR_EH_CONTINUATION OFF)
 
-    find_package(LLVM REQUIRED CONFIG PATHS $ENV{LLVM_CMAKE_CONFIG})
+    # Prefer the version specified in LLVM_CMAKE_CONFIG over the "system" installed version
+    find_package(LLVM REQUIRED CONFIG PATHS $ENV{LLVM_CMAKE_CONFIG} NO_DEFAULT_PATH)
+    find_package(LLVM REQUIRED CONFIG)
```

Comment:
Does this mean we will search for the system LLVM install too? That would not be correct.

Reply:
It does, removed.